pkg/flowkit/state.go

```diff
@@ -88,12 +107,12 @@ func (p *State) SaveEdited(paths []string) error {
 		}
 	}
 
-	return p.Save(paths[0])
+	return p.Save(paths[0], p.accountLocations)
 }
 
 // Save saves the project configuration to the given path.
-func (p *State) Save(path string) error {
-	p.conf.Accounts = accountsToConfig(*p.accounts)
+func (p *State) Save(path string, accountLocations map[string]string) error {
```

Comment:
this shouldn't be an input arg but can just be accessed from the state reference like `p.accountLocations` in the method.